opencti-platform/opencti-front/src/private/components/settings/groups/Group.tsx

```diff
@@ -173,6 +189,53 @@ const Group = ({ groupData }: { groupData: Group_group$key }) => {
     (group.default_marking ?? []).find((d) => d.entity_type === 'GLOBAL')
       ?.values ?? [],
   );
+
+  // Triggers and Digest
+  const paginationOptions = {
+    count: 25,
+    filters: [{ key: ['group_ids' as TriggerFilter], values: [group.id] }],
```

Comment:
why do you need to precise the type TriggerFilter here?

Reply:
To avoid typescript error:  `Type 'string' is not assignable to type 'TriggerFilter'.`